OtpCore/OtpAuthUri.cs

```diff
@@ -166,11 +169,15 @@ public OtpAuthUri(Uri uri)
             }
 
             if (Type == OtpType.Hotp && Parameters.ContainsKey("period"))
-                throw new ArgumentException("URI of type 'hotp' must not contain 'period' query parameter");
+                throw new ArgumentException("URI of type 'hotp' must not contain 'period' query parameter",
+                    nameof(uri));
             if (Parameters.ContainsKey("period"))
             {
                 if (!int.TryParse(Parameters["period"], out var period))
                     throw new ArgumentException("URI digits query parameter must be numeric", nameof(uri));
```

Comment:
Hmm, missed this from before, but this error message is incorrectly referring to the `digits` parameter, when the code is actually checking the `period` parameter. It appears to be just cut and paste from above and not updated.

Reply:
@Kevin-Andrew I resolved this in the main branch today.  The message will be corrected in the next version release.